bindings/modbus/mapping.ttl

```diff
@@ -0,0 +1,56 @@
+@prefix : <https://www.w3.org/2019/wot/modbus/mappings#> .
+@prefix om: <http://www.ontology-of-units-of-measure.org/resource/om-2/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix schema: <http://schema.org/> .
+@prefix binding: <https://www.w3.org/2019/wot/binding#> .
```

Comment:
from today's TD call: we should clearify if this is needed. Maybe the SHACL file from the TD can be also be used.

Reply:
Now we are using SHACL.